src/components/FooterV2.js

```diff
@@ -0,0 +1,252 @@
+import React, { useCallback } from 'react'
+import PropTypes from 'prop-types'
+import IconButton from '@material-ui/core/IconButton'
+import FacebookIcon from '@material-ui/icons/Facebook'
+import InstagramIcon from '@material-ui/icons/Instagram'
+import TwitterIcon from '@material-ui/icons/Twitter'
+import Typography from '@material-ui/core/Typography'
+import SvgIcon from '@material-ui/core/SvgIcon'
+import redirect from 'src/utils/redirect'
+import { makeStyles } from '@material-ui/core/styles'
+import logo from 'src/img/logo-with-text-white.svg'
+import GoogleChrome from 'mdi-material-ui/GoogleChrome'
+import MicrosoftEdge from 'mdi-material-ui/MicrosoftEdge'
+import AppleSafari from 'mdi-material-ui/AppleSafari'
+import {
+  chromeExtensionURL,
+  edgeExtensionURL,
+  safariExtensionURL,
+  adblockerWhitelistingURL,
+  contactUsURL,
+  externalHelpURL,
+  facebookPageURL,
+  financialsURL,
+  instagramPageURL,
+  jobsURL,
+  privacyPolicyURL,
+  teamURL,
+  termsURL,
+  twitterPageURL,
+  tiktokPageURL,
+} from 'src/utils/navigation'
+import Link from 'src/components/Link'
+const useStyles = makeStyles((theme) => ({
+  logoContainer: { flex: 1, display: 'flex', flexDirection: 'row' },
+  backgroundContainer: {
+    background: theme.palette.primary.main,
+    paddingBottom: 20,
+    display: 'flex',
+    justifyContent: 'center',
+  },
+  contentRow: {
+    display: 'flex',
+    width: '80%',
+    flexWrap: 'wrap',
+  },
+  columnOne: {
+    marginTop: theme.spacing(6.5),
+    display: 'flex',
+    flexDirection: 'column',
+    color: '#fff',
+    minWidth: theme.spacing(36),
+    width: '36%',
+  },
+  columnTwo: {
+    marginTop: theme.spacing(6.5),
+    width: theme.spacing(60),
```

Comment:
Edit to this: I think all footer content (minus future doodle) should stack to the left. Want to avoid this awkward spacing:
![Screen Shot 2021-11-09 at 1 36 42 PM](https://user-images.githubusercontent.com/5924325/140984382-ca2c7697-397b-4cf8-b1c6-3eef8482c39d.png)

So just flexbox without % width would probably work best.